File: js/src/entity/SwaggerEntity.js

const { inspect } = require('node:util')

const { ThesmsworksEntityBase } = require('../ThesmsworksEntityBase')


// TODO: needs Entity superclass
class SwaggerEntity extends ThesmsworksEntityBase {

  constructor(client, entopts) {
    super(client, entopts)
    this.name = 'swagger'
    this.name_ = 'swagger'
    this.Name = 'Swagger'
  }


  make() {
    return new SwaggerEntity(this._client, this.entopts())
  }







}


module.exports = {
  SwaggerEntity
}

File: dart/lib/entity/SwaggerEntity.dart

// ignore_for_file: non_constant_identifier_names

import 'dart:async';

import '../ThesmsworksEntityBase.dart';
import '../utility/ErrUtility.dart';

// Typed models: see ../ThesmsworksTypes.dart (Swagger and the per-op request/match types).
class SwaggerEntity extends ThesmsworksEntityBase {
  SwaggerEntity(dynamic client, dynamic entopts) : super(client, entopts) {
    name = 'swagger';
    name_ = 'swagger';
    Name = 'Swagger';
  }

  SwaggerEntity make() {
    return SwaggerEntity(client, entopts());
  }





}
